test/e2e/set/localdriver/upstream.go

```diff
@@ -6,10 +6,10 @@ import (
 	"context"
 
 	g "github.com/onsi/ginkgo/v2"
-	"github.com/scylladb/k8s-local-volume-provisioner/pkg/driver"
+	"github.com/scylladb/local-csi-driver/pkg/driver"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/api/storage/v1"
 	storagev1 "k8s.io/api/storage/v1"
+	v1 "k8s.io/api/storage/v1"
```

Comment:
aliasing `v1` with `v1` isn't needed
(this line shouldn't need touching based on the previous value https://github.com/scylladb/local-csi-driver/pull/56/files#diff-09b8c92c3407ce15738d1d7a1c08f6a4a8f71f1bd9ecd397d4037230c9b2e98dL11)
```suggestion
	"k8s.io/api/storage/v1"
```